pages/api/utils/pagePropsHelpers.ts

```diff
@@ -26,9 +26,9 @@ export const loginRedirect = (
 });
 
 // Redirect back to the dashboard if the user isn't an admin
-export const enforceAdmin: GetServerSideProps<PagePropsWithSession> = async (
+export const enforceAdmin = async (
```

Comment:
Sorry if I wasn't clear the types on `enforceAdmin` and `ensureSessionAnddAccountList` were finally and probably better how they were because they include the `session` on the props. I was just wanting to add some kind of return type to `handleUnderscoreAccountListRedirect`.

Reply:
Because I changed `handleUnderscoreAccountListRedirect ` I also had to update the other functions as they had TS errors.